### DISCUSSION TITLE
Feature Request – Integration with ScribeHow

### DISCUSSION THREAD

**Discussion by Loriel-S-Parloa** [GitbookIO/community#437] [Feature Requests]
I'd like to suggest the addition of [ScribeHow](https://scribehow.com/) as an integration feature within GitBook. While I understand that Guideflow serves a similar purpose, many individuals and organizations, including myself, rely on ScribeHow due to its widespread use.

The issue we encounter is that ScribeHow documentation cannot be incorporated into GitBook. Since ScribeHow provides an embed snippet rather than a URL address, we are unable to integrate it.

Adding ScribeHow as an official integration would greatly enhance the usability and flexibility of GitBook for a broader user base.

Thank you for considering this feature request.

**Comment by addisonschultz** [comment]
Hey @Loriel-S-Parloa! Do you have an example of what SribeHow shareable links look like? Can you share it in this thread? I'd like to test something on my end.

**Comment by Loriel-S-Parloa** [comment]
Hi @addisonschultz, sure!

This is a shareable link: 
https://scribehow.com/page/Untitled__OTVMUGEwSWWfF-_sJ6IHWA

This is an Embed: 
`<iframe src="https://scribehow.com/page-embed/Untitled__OTVMUGEwSWWfF-_sJ6IHWA" width="100%" height="640" allowfullscreen frameborder="0"></iframe>`

Thanks!

**Comment by addisonschultz** [comment]
Hi @Loriel-S-Parloa! Just tested it, and it unfortunately doesn't look like [the embedding service](https://iframely.com/domains) we use supports this out of the box. 

That being said, it still would be possible, but an integration would need to be built for this. We have a guide on how you can do that here! 

https://developer.gitbook.com/getting-started/guides/create-a-custom-unfurl-action-for-your-integration

Hopefully this helps for now

Addison

**Comment by jhoney124** [comment]
Hopefully, this helps for now..

**Comment by addisonschultz** [comment]
@jhoney124 @Loriel-S-Parloa I found some time and made an integration for ScribeHow 😄 

You can install it using this link! 

https://app.gitbook.com/integrations/scribe-how

After installing the integration, you'll just need to paste in the URL from a ScribeHow project. Let me know if you have any questions!
